### PR TITLE
Search bar  style upgrade.

### PR DESCRIPTION
**Pull request by prwlr84**

### CODE REVIEW

File: app/assets/stylesheets/pages/_home.scss [hunk 1]
```diff
@@ -20,17 +20,50 @@ body {
...
   font-family: 'Nanum Gothic Coding', monospace;
 }
 
-// width of the search bar on the homepage
-#mainSelect {
-  width: 25%;
+
+
+
+form {
```
Comment (tiagoenes) [inline app/assets/stylesheets/pages/_home.scss]: It's gonna change all the forms of the app

File: app/assets/stylesheets/pages/_home.scss [hunk 2]
```diff
@@ -20,17 +20,50 @@ body {
...
+  display: flex;
+  justify-content: center;
+  width: 50%;
+  border-radius: 25px;
+  overflow: hidden;
+  border: 1px solid;
+  background-color: white;
+
+  .form-group{
```
Comment (tiagoenes) [inline app/assets/stylesheets/pages/_home.scss:27]: it's gonna change all the form groups of the app

File: app/assets/stylesheets/pages/_home.scss [hunk 3]
```diff
@@ -20,17 +20,50 @@ body {
...
+    display: inline-block;
+    margin: 0;
+    flex: 2;
+    input{
```
Comment (tiagoenes) [inline app/assets/stylesheets/pages/_home.scss:31]: it's gonna change all the inputs inside form groups

File: app/assets/stylesheets/pages/_home.scss [hunk 4]
```diff
@@ -20,17 +20,50 @@ body {
...
+      width:100%;
+      height: 100%;
+      padding: 0;
+      border: 0;
+      border-radius: 25px;
+      padding-left: 25px;
+    }
+
+    input:focus, textarea:focus {
```
Comment (tiagoenes) [inline app/assets/stylesheets/pages/_home.scss:40]: same here

**Review by tiagoenes — CHANGES_REQUESTED** [review]
You have to create specific classes for your tags because you are using tag used genericly

**Comment by prwlr84** [comment]
added id should be better now

**Pull request merged**